app/poros/quote.rb

```diff
@@ -0,0 +1,9 @@
+class Quote
+  attr_reader :author,
+              :category
+  
+  def initialize(attributes)
+    @author = attributes[:author]
+    @category = attributes[:tags]
```

Comment:
Currently your tags are in an array. Can you think of a way that you could get them out of an array and get a comma separated string instead? :)